pennylane/qchem/openfermion_obs.py

```diff
@@ -28,7 +28,7 @@
 def import_of():
     """Import openfermion and openfermionpyscf."""
     try:
-        # pylint: disable=import-outside-toplevel, unused-import
+        # pylint: disable=import-outside-toplevel, unused-import, multiple-imports
```

Comment:
Linting: although unrelated, likely Codefactor would fail due to changes in this file, hence this line added.